### PR TITLE
Add dX_2000A/dY_2000A access to the IERS_A table

### PR DESCRIPTION
**Pull request by mcbeth**
As stated above, it adds interpolation of the dX_2000A and dY_2000A columns to the IERS table.  This allows for more accurate conversion between the coordinate frames that use the Celestial Intermediate Pole.  The use of these new columns are not currently wired into any of the rest of the coordinate transform code.

### CODE REVIEW

**Comment by bsipocz** [comment]
This needs review and a rebasing. Could either ofyou @adrn @mhvk or @eteq have a look?

**Comment by mhvk** [comment]
@mcbeth - sorry that this slipped by completely when you submitted the request. The code looks all OK, though it should have a few tests just to be sure (in `iers/tests/test_iers.py` - just add to what's there for UT1 and PM_x, PM_y.

Also, where you planning to help actually use this in the coordinate transformation? That would obviously be great, but if you don't have time, that is fine too - then we should just open an issue noting that this is available and could be used.

**Comment by mcbeth** [comment]
On Thu, Nov 30, 2017 at 01:34:03PM +0000, Marten van Kerkwijk wrote:
>    [1]@mcbeth - sorry that this slipped by completely when you submitted the
>    request. The code looks all OK, though it should have a few tests just to
>    be sure (in iers/tests/test_iers.py - just add to what's there for UT1 and
>    PM_x, PM_y.

I can do that.  Getting this merged will be immensely helpful.

>    Also, where you planning to help actually use this in the coordinate
>    transformation? That would obviously be great, but if you don't have time,
>    that is fine too - then we should just open an issue noting that this is
>    available and could be used.

I'd like to help.  I haven't actually dug into that portion of code recently, as I was on a tight schedule and ended up abusing the hidden erfa bindings.

Jeff

-- 
"The man who does not read good books has no advantage over 
 the man who cannot read them."
 -- not Mark Twain, maybe a southen librarian in 1910

**Comment by mhvk** [comment]
@mcbeth - great! Do ping me when you've added the tests (and I fear a rebase is necessary as well).

**Comment by bsipocz** [comment]
(@mcbeth - if you need help with the rebasing, I can walk you through it at the astropy slack, just ping me there)

File: CHANGES.rst [hunk 1]
```diff
@@ -151,6 +151,11 @@ astropy.utils
 
 - ``JsonCustomEncoder`` is expanded to handle ``Quantity`` and ``UnitBase``.
   [#5471]
+- ``astropy.utils``
```
Comment (eteq) [inline CHANGES.rst]: This is already in the utils section so doesn't require this added part.

File: CHANGES.rst [hunk 2]
```diff
@@ -151,6 +151,11 @@ astropy.utils
...
+
+  - Added ``dcip_xy`` method to IERS
+	The data for the CIP offsets is now available for use
+	in coordinate frame conversion
```
Comment (eteq) [inline CHANGES.rst]: Also this needs ``[#5837]`` at the end to indicate the Pull Request it came from.

**Comment by eteq** [comment]
(note that travis and circle-ci were passing in the last commit - the X next to the last commit that wasn't me is only because of the changelog issues)

**Comment by mcbeth** [comment]
On Fri, Dec 15, 2017 at 01:43:10AM +0000, Erik Tollerud wrote:
>    @eteq requested changes on this pull request.
> 
>    This looks good aside from the problems with the changelog. [1]@mcbeth
>    I'll go ahead and just fix these now so we can merge, but you might want
>    to look at the inline comments I've left just so you can see what I mean.

Looks great.  I was confused how I would get a PR# before submitting a PR :)
Now I see that I should have just submitted, gotten the PR, and then committed again.

Jeff

-- 
"The man who does not read good books has no advantage over 
 the man who cannot read them."
 -- not Mark Twain, maybe a southen librarian in 1910

**Comment by bsipocz** [comment]
Yes, that's a bit tricky and easiest done in a new commit once the PR is open. (Otherwise if you need to do it often, we have a script for that https://github.com/astropy/astropy-tools/blob/master/next_pr_number.py, though that assumes no one opens an issue or PR after your run it and open your PR).

**Pull request merged**